src/dlsproto/client/legacy/internal/helper/GroupRequest.d

```diff
@@ -88,3 +91,30 @@ public template GroupRequest ( Request )
         GroupRequest;
 }
 
+/*******************************************************************************
+
+    Instantiates GroupRequest instance wrapped around Request instance. Used
```

Comment:
FWIW, it would be more grammatical to write "Instantiates a GroupRequest instance wrapped around the provided Request instance".
But this LGTM.

Reply:
Thanks, I'll reword (you may notice a complete disregard for articles in my text).